### PR TITLE
Bump zwave-js to 10.0.1 and zwave-js-server to 1.22.1

### PR DESCRIPTION
**Pull request by raman325**
Changelogs:
- `zwave-js-server`:
  - https://github.com/zwave-js/zwave-js-server/releases/tag/1.22.0
  - https://github.com/zwave-js/zwave-js-server/releases/tag/1.22.1
- `zwave-js`:
  - https://github.com/zwave-js/node-zwave-js/releases/tag/v10.0.0
  - https://github.com/zwave-js/node-zwave-js/releases/tag/v10.0.1

### CODE REVIEW

**Comment by raman325** [comment]
Moving to draft because we should bump zwave-js-server to 1.22.1 and zwave-js to 10.0.1, the former of which is awaiting a PR merge and release

File: zwave_js/build.yaml [hunk 1]
```diff
@@ -9,5 +9,5 @@ codenotary:
   signer: notary@home-assistant.io
   base_image: notary@home-assistant.io
 args:
-  ZWAVEJS_SERVER_VERSION: 1.21.0
-  ZWAVEJS_VERSION: 9.6.2
+  ZWAVEJS_SERVER_VERSION: 1.22.0
+  ZWAVEJS_VERSION: 10.0.0
```
Comment (MartinHjelmare) [inline zwave_js/build.yaml:13]:
```suggestion
  ZWAVEJS_SERVER_VERSION: 1.22.1
  ZWAVEJS_VERSION: 10.0.1
```

**Review by MartinHjelmare — APPROVED** [review]
I've tested building the add-on locally and running it with the integration. Thanks!

File: zwave_js/build.yaml (hunk 1 above)
Comment (raman325) [reply zwave_js/build.yaml]: 🤦

**Pull request merged**